inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/fieldtype/FieldTypeMappingReader.java

```diff
@@ -59,7 +64,10 @@ public class FieldTypeMappingReader implements Serializable {
     @Getter
     protected final String streamType;
     @Getter
-    protected final Map<String, String> FIELD_TYPE_MAPPING_MAP = Maps.newHashMap();
+    protected final Map<String, String> STREAM_TO_SINK_FIELD_TYPE_MAPPING_MAP = Maps.newHashMap();
+
+    @Getter
+    protected final Map<String, String> SOURCE_TO_SINK_FIELD_TYPE_MAPPING_MAP = Maps.newHashMap();
```

Comment:
why are these two variables named in the upper case?

Reply:
Fixed.